project/plugins.sbt

```diff
@@ -9,7 +9,7 @@ resolvers += "Bintray Jcenter" at "https://jcenter.bintray.com/"
 addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.3.8")
 //#sbt-multi-jvm
 
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
+addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.0")
```

Comment:
That's the version we'll be able to use on sbt 1 too right?

Reply:
Yes, that was indeed the trigger to finally push this (though I also like the new `NewlineAtEndOfFile` setting :) )